### ISSUE TITLE
Referee report from Sept 14, 2022

### ISSUE THREAD

**Issue by gully** [gully/blase#52]
We got our referee report back and it is really positive, with mostly minor revisions.  Yay!  🥳  Here is a paraphased and editorialized version of some of the feedback from the report, plus some themes that the report may have indirectly catalyzed.  I'll add check boxes for the purposes of addressing each line item:

First round:

- [x] Revise running title.  It's currently "Transfer Learning with Blasé"
- [x] Transfer learning does not appear to be defined in the text of the paper.  (Maybe we should add it to Section 7.1 "Surrogate modeling" sub paragraph?). The phases "physics-informed ML" and "simulation intelligence" could slot in here as well.  Basically, we what to describe how an ML may describe this approach, and transfer learning is only partly accurate.  
- [x] Define the use of "clone" (we appear to just use it without explanation)
- [x] (aside) Consider citing more work from the Cannon 
- [x] Expand and revise Section 8.4 about previously unknown spectral lines
- [x] Why PyTorch? Compared to TensorFlow / Jax / Theano
- [x] Related: Discuss framework incompatibility and [vendor lock-in](https://en.wikipedia.org/wiki/Vendor_lock-in).  
- [x] Consider discussing the approach in terms of chemically peculiar stars (e.g. HgMn stars)

**Comment by gully** [comment]
Minor Comments:
- [ ] TensorFlow -> CamelCase. 
- [ ] The \odot elementwise product notation is slightly nonstandard and should be better defined
- [ ] Better explanation of the edge buffers (page 5 top)
- [ ] Describe the `scipy` thresholding finding algorithm and the definition of prominence 
- [ ] Rephrase the Jacobian section to discuss dimensionality-- scalar-to-vector
- [ ] Focus the discussion on gradient-based versus non-gradient based optimization (instead of the current "conventional" versus "SGD").
- [ ] Use the terminology "full batch gradient descent" and not "stochastic" 
- [ ] l 344 paragraph - Better discussion of why the pseudoVoigt has bad fits in the line cores.
- [ ] Provide a link to the Hogg Telluric Hack Week Slides
- [ ] Captialization of elements is funky
- [ ] l 373, 382, 423 use \noindent after equation
- [ ] standardize italics or texttt for blasé
- [ ] l 406 "mathematically speaking, multiplication and convolution do not commute" should have a qualifier for vector operations (scalar does commute)
- [ ] l 590 - corollary mis-spelled
- [ ] l 619 - "reverse engineer" could be replaced with just "infer"
- [ ] The magnetific field discussion in Sec 8.6 ends suddenly

**Comment by gully** [comment]
OK, almost done with this report! Only two more pieces left: 

- l 344 paragraph - Better discussion of why the pseudoVoigt has bad fits in the line cores.
- The magnetific field discussion in Sec 8.6 ends suddenly
